kclvm/ast/src/ast.rs

```diff
@@ -1632,6 +1639,9 @@ impl ToString for Type {
                         w.push_str(&format!("\"{}\"", v.replace('"', "\\\"")));
                     }
                 },
+                Type::Function(_) => {
+                    w.push_str("function");
```

Comment:
At this point, it is necessary to output the `(x) -> y` source form, as the AST printer needs to be used, and corresponding test cases can be added